src/main/java/com/spring/app/CountryLevel.java

```diff
@@ -1,5 +1,10 @@
 package com.spring.app;
 
+/**
+ * CountryLevel
+ * Contains a living level in country
+ * @author lyubov
+ */
 public enum CountryLevel {
```

Comment:
Name is not consistent, From the first grasp I don't understand what Country Level is, is it like mmorpg ? Choose the right one , EconomyLevel etc..

Reply:
changed